### PR TITLE
Football Leagues React Capstone

### PR DESCRIPTION
**Pull request by shahira-sadat**
In this PR I did the following 👇🏻:

-  Setup the working environment
-  Installed the required dependencies
-  Setup `redux store` and `reducer slice`
-  Created the components
-  Styled the components
-  Added test and snapshot
-  Updated the `README.md` file

[Project Presentation Video](https://drive.google.com/file/d/1zZb6jTosyjxTOyTocpVO5PmEtIG7mndZ/view?usp=sharing)

### CODE REVIEW

**Review by Reem-lab — CHANGES_REQUESTED** [review]
# Hi @shahira-sadat, 👋🏻👋🏻

Good job so far! 🔥🔥💯💯💯🚀🚀
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights ✨✨
📌 No Linters error ✅
📌 Descriptive pull request✅
📌 Great working on the Design ✅
📌 Good commit messages ✅
📌 Professional README ✅


## Required Changes ♻️
_Check the comments under the review._
- [ ] I noticed that you forget to record a loom  video that explain and showcase your project and since it is one of the requirements so kindly record it and add the link to your PR 
image


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by shahira-sadat** [comment]
> # Hi @shahira-sadat, 👋🏻👋🏻
> Good job so far! 🔥🔥💯💯💯🚀🚀 There are some issues that you still need to work on to go to the next project but you are almost there!  [ 68747470733a2f2f6d65646961302e67697068792e636f6d2f6d656469612f333151333645484a3743355a6f494d7737392f67697068792e676966 ](https://camo.githubusercontent.com/8c8e2827bb31e1c10e7f8607b0c89f468ffd62651515fafce1fb51b5a8f80779/68747470733a2f2f6d65646961302e67697068792e636f6d2f6d656469612f333151333645484a3743355a6f494d7737392f67697068792e676966) [ ](https://camo.githubusercontent.com/8c8e2827bb31e1c10e7f8607b0c89f468ffd62651515fafce1fb51b5a8f80779/68747470733a2f2f6d65646961302e67697068792e636f6d2f6d656469612f333151333645484a3743355a6f494d7737392f67697068792e676966)
> 
> ## Highlights ✨✨
> 📌 No Linters error ✅ 📌 Descriptive pull request✅ 📌 Great working on the Design ✅ 📌 Good commit messages ✅ 📌 Professional README ✅
> 
> ## Required Changes ♻️
> _Check the comments under the review._
> 
> * [ ]  I noticed that you forget to record a loom  video that explain and showcase your project and since it is one of the requirements so kindly record it and add the link to your PR
>   image
> 
> ## Optional suggestions
> _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
> 
> Cheers and Happy coding!👏👏👏
> 
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.
> 
> **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
> 
> _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

Added Link to video ✅

**Comment by shahira-sadat** [comment]
Added Link to video ✅

File: README.md [hunk 1]
```diff
@@ -35,7 +44,7 @@ To get a local copy up and running follow these simple example steps.
```
Comment (Bondok6) [inline README.md:47]: - [OPTIONAL] I suggest adding one more command in your README file for testing `npm run test` 👌

File: README.md [hunk 2]
```diff
@@ -56,3 +65,5 @@ Give a ⭐️ if you like this project!
 ## Acknowledgments
 
 - [Microverse](https://www.microverse.org/)
+- Thanks to [Ali Jendoubi](https://github.com/ichala) for making the Football Leagues API that used in this project.
+- Thanks to [Nelson Sakwa on Behance.](https://www.behance.net/sakwadesignstudio) for App design template.
```
Comment (Bondok6) [inline README.md:72]: - [OPTIONAL] I suggest adding the `MIT.md` file to your project 👌, Here is the [Micorverse Template](https://github.com/microverseinc/readme-template/blob/master/MIT.md), and remember to customize it 😉

> The MIT license gives users express permission to reuse code for any purpose, sometimes even if the code is part of proprietary software. As long as users include the original copy of the MIT license in their distribution, they can make any changes or modifications to the code to suit their own needs. 💯

**Review by Bondok6 — APPROVED** [review]
# STATUS: APPROVED :green_circle: :clap: 

## Hi @shahira-sadat 🙋‍♂️, 
Your project is complete!  There is nothing else to say other than... Great job 👏 
Congratulations! 🎉 

TO HIGHLIGHT:

- Your project is professional ✅
- Your repo is professional ✅
- Well structured files ✅
- No Linter errors ✅


Now you can merge this branch and move on :1st_place_medal: 
HAPPY CODING AND GOOD LUCK WITH THE COMMING ONES :crossed_fingers: :four_leaf_clover: 




## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by shahira-sadat** [comment]
Implemented the optional requirements ✅

**Pull request merged**